### DISCUSSION TITLE
Microsoft hyper v virtualization

### DISCUSSION THREAD

**Discussion by Pascar85** [Security-Onion-Solutions/securityonion#10424] [Unsupported Versions]
Hi.. I was trying to install security onion on my test Microsoft hyper v platform since yesterday but its seems its not supported I was trying to test the new 2.4 release on rocky 9. but ended in an error after a long install. then I tried with the 2.3 iso, it started of fine at first then it also ended in an error. is the platform not supported on hyper v? 

Best Regards

**Comment by dougburks** [comment]
I believe others have successfully installed Security Onion in Hyper-V. You can try searching the discussions here for more info (but please do not reply to old discussions).

If you'd like further assistance with the errors you received, please include the full error message and detailed information about your system specs and the options you chose when installing.
